vector/src/main/java/im/vector/app/features/home/room/detail/timeline/helper/TimelineSettingsFactory.kt

```diff
@@ -48,12 +53,12 @@ class TimelineSettingsFactory @Inject constructor(private val userPreferencesPro
         }
     }
 
-    private fun List<String>.filterDisplayableTypes(): List<String> {
-        return filter { type ->
-            when (type) {
-                EventType.STATE_ROOM_MEMBER -> userPreferencesProvider.shouldShowRoomMemberStateEvents()
-                else                        -> true
-            }
+    private fun List<String>.filterDisplayableTypes(): List<EventTypeFilter> {
```

Comment:
Maybe change the name of the method to reflect mapping?

Reply:
Done.